x/asset/client/cli/flags.go

```diff
@@ -3,13 +3,27 @@ package cli
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+	"strconv"
+	"strings"
 )
 
 const (
-	flagLiquidationRatio       = "liquidation-ratio"
-	flagName                   = "name"
-	flagDenom                  = "denom"
-	flagDecimals               = "decimals"
+	flagLiquidationRatio        = "liquidation-ratio"
+	flagName                    = "name"
+	flagDenom                   = "denom"
+	flagDecimals                = "decimals"
+	flagCollateralWeight        = "collateralWeight"
```

Comment:
collateral-weight
liquidation-threshold
is-bridged-asset , lets follow similar naming style across. @cgsingh33 @pratikasr @rajrohit10